app/models/user.rb

```diff
@@ -90,4 +90,8 @@ def self.from_omniauth(auth)
       user.password = Devise.friendly_token[0, 20]
     end
   end
+
+  def owns_group?(group)
+    owned_groups.include?(group)
+  end
```

Comment:
Tbh, now that I think about it: I don't like this method, as it doesn't really add anything of value. Do you mind removing it?

Reply:
I think it makes the intent clearer of what the dev wants to express. What is it you do not like about the method?

Reply:
I feel like this method doesn't increase the readability by much and is just one more indirection. But I also see your point, so feel free to keep it.